gqlalchemy/graph_algorithms/query_modules.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) 2016-2022 Memgraph Ltd. [https://memgraph.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Tuple, List
+
+QM_KEY_NAME = "name"
+QM_KEY_VALUE = "value"
+QM_KEY_DEFAULT = "default"
+QM_KEY_TYPE = "type"
+
+QM_FIELD_NAME = "name"
+QM_FIELD_IS_EDITABLE = "is_editable"
+QM_FIELD_IS_WRITE = "is_write"
+QM_FIELD_PATH = "path"
+QM_FIELD_SIGNATURE = "signature"
+
+
+class QueryModule:
+    """Class representing a single MAGE query module."""
+
+    def __init__(self, module_dict: Dict) -> None:
+        arguments, returns = parse_query_module_signature(module_dict[QM_FIELD_SIGNATURE])
+
+        self.name = module_dict[QM_FIELD_NAME]
+        self.is_editable = module_dict[QM_FIELD_IS_EDITABLE]
+        self.is_write = module_dict[QM_FIELD_IS_WRITE]
+        self.path = module_dict[QM_FIELD_PATH]
+        self.signature = module_dict[QM_FIELD_SIGNATURE]
+        self.arguments = arguments
+        self.returns = returns
+
+    def __str__(self) -> str:
+        return self.name
+
+    def set_argument_values(self, **kwargs) -> None:
+        """Set values for QueryModule arguments so the module can be called.
+
+        Kwargs:
+            Named arguments in self.arguments.
+
+        Raises:
+            KeyError: Passed an argument not in the self.arguments list.
+        """
+        for argument_name in kwargs.keys():
+            has_arg = False
+            for argument_dict in self.arguments:
+                if argument_dict[QM_KEY_NAME] == argument_name:
```

Comment:
use `in` operator here. This for loop will run more times than necessary(what if argument name is found in the first place?)